testsuite/features/support/env.rb

```diff
@@ -144,9 +150,12 @@ def capybara_register_driver
       attach "#{Time.at(@scenario_start_time).strftime('%H:%M:%S:%L')} - #{Time.at(current_epoch).strftime('%H:%M:%S:%L')} | Current URL: #{current_url}", 'text/plain'
     rescue StandardError => e
       warn e.message
+      register_driver
```

Comment:
Those changes don't seem related to the PR.

Reply:
That's a leftover of the bigger refactor I made. This was necessary to recover properly when we run tests using parallel_tests gem, and some scenario fails.

I can revert it, and keep in another branch for now, still we gonna need it.

Reply:
Move into qe-refactor-enabling-parallel-tests branch